website/content/en/highlights/2022-05-24-vrl-encryption.md

```diff
@@ -0,0 +1,97 @@
+---
+date: "2022-05-24"
+title: "New encrytion functions for VRL"
+description: "Encrypt and decrypt data in VRL"
+authors: ["jszwedko"]
+pr_numbers: []
+release: "0.22.0"
+hide_on_release_notes: false
+badges:
+  type: enhancement
+---
+
+VRL now contains facilicities for for encrypting and decrypting field values using [AES][AES] encryption with user
```

Comment:
> [AES][AES]
Is this supposed to be a link?

Reply:
Ah, good catch, I used `aes` at the bottom for the link.